CHANGELOG-internal.asciidoc

```diff
@@ -0,0 +1,16 @@
+// Use these for links to issue and pulls. Note issues and pulls redirect one to
+// each other on Github, so don't worry too much on using the right prefix.
+:issue: https://github.com/elastic/beats/issues/
+:pull: https://github.com/elastic/beats/pull/
+
+This internal changelog is intended to cover the major breaking changes to the interal API's in the official Beats to make it for community Beats easier to follow the changes. Only the major changes will be covered in this changelog which are not all changes to the API's. Each breaking change added here should have an explanation on how other Beats should be migrated.
```

Comment:
I think "developer" (or maybe "api") rather than "internal" would be a better name. When I hear "internal" I'm thinking of an API that only a core developer would care about, not something that would affect a person developing their own beat, processor, output, codec, filebeat input, heartbeat monitor, metricbeat module, packetbeat protocol,  or autodiscovery appender/builder/provider (I did a search to see what parts of beats have pluggable interfaces).

Reply:
I started with api initially but then realised it could be more then just the API that changes. For example if something in the packaging or generation of the beat changes it should be there too.

I like the developer proposal.